test/check-application

```diff
@@ -555,6 +559,75 @@ class TestApplication(testlib.MachineCase):
         b.wait_text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(3)", "")
         b.wait_text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(4)", "")
 
+    def testCheckpointRestore(self):
+        b = self.browser
+        m = self.machine
+
+        # Set machine to cgroup1 and reboot
+        self.execute(True, 'grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"; reboot')
+        m.wait_reboot()
+
+        self.login_and_go("/podman", authorized=True, superuser=True)
+        b.wait_present("#app")
+        self.filter_containers('all')
+
+        # Run a container
+        self.execute(True, "podman run -dit --name swamped-crate busybox sh; podman stop swamped-crate")
```

Comment:
Does checkpoint/restore work only for root containers and not for rootless?
If so, then we should check that these actions don't show up for rootless containers. If it works, then this test should have two modes System/User. (See how for example testBasic{System,User} is done)

Reply:
Checkpoint/restore in Podman is currently supported only on root containers. The action actually shows up for user containers and causes an error - I'll implement a check that will hide the buttons for user containers and a test for it.